### PR TITLE
Javascript I

### PR DESCRIPTION
**Pull request by JacobWilliams90**

### CODE REVIEW

**Comment by JacobWilliams90** [comment]
"Moved the push out of curly braces" Should be "Moved the Sort out of the curly braces"

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,57 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car = inventory[32]
```
Comment (liam-mei) [inline assignments/arrays.js:66]: I did the same thing the first time around, but I believe we're supposed to loop over the inventory array to find the id, and not use the inventory's index.  I'm sure you know how to do it that way anyways though

File: assignments/function-conversion.js [hunk 2]
```diff
@@ -5,26 +5,42 @@
...
 
+let myFunction = () => console.log("Function Was Invoked!");
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => param;
```
Comment (liam-mei) [inline assignments/function-conversion.js:16]: small note, if there's only one param, you can leave out the parens.  Either way works.  Just additional syntax that is valid

**Review by liam-mei — APPROVED** [review]

**Comment by liam-mei** [comment]
Great Job completing the project including part of object stretch.

Score 3

File: assignments/objects.js [hunk 3]
```diff
@@ -49,16 +85,31 @@ const example = {
...
+  child: {
+    name: "George",
+    age: 50,
+    grandchild: {
+      name: "Sam",
+      age: 30,
+    }
+  },
+  sayname: function(){
+    return "Hello, My name is " + this.name;
```
Comment (liam-mei) [inline assignments/objects.js:100]:
```
const parent = {
  name: "Susan",
  age: 70,
  speak: function(){console.log(`my name is ${this.name}`)},
  child: {
    name: "George",
    age: 50,
    speak: function(){console.log(`my name is ${this.name}`)},
    grandchild: {
      name: "Sam",
      age: 30,
      speak: function(){console.log(`my name is ${this.name}`)}
    }
  },
};

parent.speak()
parent.child.speak()
parent.child.grandchild.speak()
```

**Review by liam-mei — APPROVED** [review]

**Pull request merged**